show_ligand_interactions.py

```diff
@@ -0,0 +1,43 @@
+#show_ligand_interactions v.1.0
+# author: Thomas Evangelidis, 2019
+# License: BSD-2-Clause
+
+from pymol import cmd, util
+
+def show_ligand_interactions(recsel="not hetatm", ligsel="hetatm", cuttof=5):
+    """
+DESCRIPTION
+
+    Visualize interactions between receptor and ligand.
+
+ARGUMENTS
+
+    recsel = string: atom selection of the receptor {default: "not hetatm"}
+
+    ligsel = string: atom selections of the ligand {default: "hetatm"}
+
+    cutoff = float: show as sticks all receptor residues within this distance from the ligand {default: 5.0}
+    """
+    cmd.select('ligand', ligsel)
+    cmd.select('receptor', recsel)
+    cmd.bg_color('white')
+    cmd.show_as('cartoon')
+    cmd.show_as('sticks', 'hetatm')
+    cmd.set('cartoon_transparency', 0.2)
+    cmd.spectrum(selection=recsel+" or "+ligsel,byres=1)
+    util.cbag('not name C*')
```

Comment:
`not elem C` is more robust, e.g. wouldn't include calcium atoms

Reply:
indeed!